pkg/sdk/validation/manifest/json_remote_type_test.go

```diff
@@ -0,0 +1,151 @@
+package manifest_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"capact.io/capact/internal/cli/heredoc"
+	gqlpublicapi "capact.io/capact/pkg/hub/api/graphql/public"
+	"capact.io/capact/pkg/sdk/apis/0.0.1/types"
+	"capact.io/capact/pkg/sdk/validation/manifest"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	genericStorageSchema = heredoc.Doc(`
+	{
```

Comment:
As pointed out in https://github.com/capactio/hub-manifests/pull/66, please update the schema 🙁 And then you can update invalid schema test case, e.g. by providing schema which has invalid contextSchema shape or URL.

Reply:
Fixed, I improve the schema and add more tests:+1: